Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -133,6 +133,15 @@
             "required": false,
             "helpMarkDown": "Enabling this option will generate a test results TRX file in `$(Agent.TempDirectory)` and results will be published to the server. <br>This option appends `--logger trx --results-directory $(Agent.TempDirectory)` to the command line arguments. <br><br>Code coverage can be collected by adding `--collect \"Code coverage\"` option to the command line arguments. This is currently only available on the Windows platform."
         },
+        {
+            "name": "testRunTitle",
```

Comment:
Can you update the task version to 2.150.0

Reply:
done in new commit